data.sql

```diff
@@ -13,3 +13,29 @@ VALUES ('Pikachu', '7-1-2021', '1', 'false', '15.04')
 INSERT INTO animals
 (name, date_of_birth, escape_attempts, neutered, weight_Kg)
 VALUES ('Devimon', '12-5-2017', '5', 'true', '11')
+
+--second batch of data
+
+INSERT INTO animals
+(name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES ('Charmander', '2020=02-08', '0', 'false', '11');
+
```

Comment:
- [ ] The value for the `weight_kg` for this data is `-11` instead of `11`. Please let's fix it.